tests/scalar/test_basic.py

```diff
@@ -531,3 +531,11 @@ def test_scalar_hash_default_output_type_preference():
     del old_eq.output_types_preference  # mimic old Op
     assert new_eq == old_eq
     assert hash(new_eq) == hash(old_eq)
+
+
+def test_rtruediv():
+    from pytensor.scalar.basic import ScalarType, TrueDiv
```

Comment:
0. Use global imports
1. Test floordiv as well. 
2. Assert the output type is still a ScalarType
3. Call `y.eval()` to be sure it works.